browser/lib/transport/jsonptransport.js

```diff
@@ -1,6 +1,6 @@
 var JSONPTransport = (function() {
 	var noop = function() {};
-	var _ = window.Ably._ = {};
+	var _ = Ably._ = {};
```

Comment:
This will break the jsonp transport unfortunately, it currently relies on there being a `window.Ably` object.